glusterd2/commands/volumes/volume-create-txn.go

```diff
@@ -92,6 +92,15 @@ func populateSubvols(volinfo *volume.Volinfo, req *api.VolCreateReq) error {
 			if subvolreq.ReplicaCount != 2 || subvolreq.ArbiterCount != 1 {
 				return errors.New("for arbiter configuration, replica count must be 2 and arbiter count must be 1. The 3rd brick of the replica will be the arbiter")
 			}
+			var abriterBrickCount uint16
+			for _, b := range subvolreq.Bricks {
+				if b.Type == "arbiter" {
+					abriterBrickCount++
+				}
+			}
+			if abriterBrickCount != 1 {
```

Comment:
You can change this to `if abriterBrickCount != subvolreq.ArbiterCount`. Currently only 1 is supported, in future if more than one arbiter bricks supported then only change required in Line 92